docs/guide/index.md

```diff
@@ -1,17 +1,18 @@
 ---
 title: "Guide"
-has_children:  true
+has_children: true
 has_toc: true
 nav_order: -4
 ---
 
 [info]: this file is autogenerated
+
 # The Fuel TypeScript SDK Guide
 
 TypeScript SDK for Fuel. It can be used for a variety of things, including but not limited to:
 
 - Compiling, deploying, and testing [Sway](https://github.com/FuelLabs/sway) contracts;
-- Use the [Testnet](../providers/external-node.md) or run a local Fuel node;
+- Use the [Testnet](./providers/connecting-to-an-external-node.md) or run a local Fuel node;
 - Crafting and signing transactions with hand-crafted scripts or contract calls;
 - Generating type-safe TypeScript bindings of contract ABI methods;
 - And more. `fuels-ts` is still in active development.
```

Comment:
We actually don't need to change any files under `docs/guide`. All the contents of this folder are auto-generated by our CI. We just need to change the files under `docs/_guide`, which you have already done. Please feel free to revert the changes you made to `docs/guide/index.md` and `docs/guide/providers/connection.md`!